chat-widget/src/components/webchatcontainerstateful/WebChatContainerStateful.tsx

```diff
@@ -3,9 +3,7 @@
 import { IStackStyles, Stack } from "@fluentui/react";
 import { LogLevel, TelemetryEvent } from "../../common/telemetry/TelemetryConstants";
 import React, { Dispatch, useEffect } from "react";
-
 import { BotMagicCodeStore } from "./webchatcontroller/BotMagicCodeStore";
-import { BroadcastChannel } from "broadcast-channel";
```

Comment:
If this is not used any more, shall we remove it from package.json and yarn.lock?